otter/integration/tests/test_convergence.py

```diff
@@ -1121,13 +1129,14 @@ def test_scale_over_lb_limit(self):
             .addCallback(scale_up_to_group_max.execute)
             .addCallback(
                 group.wait_for_state,
-                MatchesAll(
-                    ContainsDict({
-                        'pendingCapacity': Equals(0),
-                        'desiredCapacity': Equals(group_max),
-                        'status': Equals("ERROR")
-                    }),
-                    HasActive(LB_max)),
+                ContainsDict({
+                    'pendingCapacity': GreaterThan(group_max - LB_max - 1),
+                    'desiredCapacity': Equals(group_max),
+                    'status': Equals("ERROR"),
+                    'active': AfterPreprocessing(
+                        len, MatchesAll(GreaterThan(LB_max - 6),
```

Comment:
I'm thinking this LB_max - 6 should either have a comment or reference an initial_offset variable to link this to the starting desired capacity in order to improve maintainability.